lib/internal/Magento/Framework/Setup/Declaration/Schema/Dto/Constraints/Reference.php

```diff
@@ -111,6 +119,16 @@ public function getOnDelete()
         return $this->onDelete;
     }
 
+    /**
+     * On update action.
+     *
+     * @return string
+     */
+    public function getOnUpdate()
```

Comment:
This is actually backward incompatible change. As for me - it's good reason to add it. @buskamuza  what do you think?

Reply:
@buskamuza any updates?

Reply:
Created issue in architecture repo https://github.com/magento/architecture/issues/310